### PR TITLE
KB-H053: allow custom generators

### PR DESCRIPTION
**Pull request by ericLemanissier**
conans.model.Generator is explicitly documented in https://docs.conan.io/en/latest/howtos/custom_generators.html

see https://github.com/conan-io/conan-center-index/pull/1759#issuecomment-711129395

### CODE REVIEW

File: hooks/conan-center.py [hunk 1]
```diff
@@ -558,6 +558,10 @@ def test(out):
     @run_test("KB-H053", output)
     def test(out):
         def _is_private_import(line):
+            full_line_allowed_list = ["from conans.model import Generator"]
+            if line in full_line_allowed_list:
+              return true
+
```
Comment (uilianries) [inline hooks/conan-center.py:564]:
```suggestion
        if conanfile.name in ["qt"]:
            out.info("'{}' is part of the allowlist.".format(conanfile.name))
            return

        def _is_private_import(line):
```

**Review by uilianries — CHANGES_REQUESTED** [review]

File: hooks/conan-center.py [hunk 2]
```diff
@@ -558,6 +558,10 @@ def test(out):
     @run_test("KB-H053", output)
     def test(out):
         def _is_private_import(line):
+            full_line_allowed_list = ["from conans.model import Generator"]
+            if line in full_line_allowed_list:
+              return true
+            
             allowed_list = ["tools", "errors"]
```
Comment (ericLemanissier) [inline hooks/conan-center.py:565]:
```suggestion            
            allowed_list = ["tools", "errors", "model.Generator"]
```
@uilianries what do you think of this one ?

Comment (uilianries) [reply hooks/conan-center.py:565]: sounds better!

**Review by uilianries — APPROVED** [review]

**Comment by ericLemanissier** [comment]
this is blocking https://github.com/conan-io/conan-center-index/pull/1759

**Comment by ericLemanissier** [comment]
sorry, for the noise, but is there something blocking on this one ? @danimtb @jgsogo

**Review by jgsogo — APPROVED** [review]

**Pull request merged**